packages/example/web/index.html

```diff
@@ -31,19 +31,31 @@
 
     <title>example</title>
     <link href="manifest.json" rel="manifest">
-    <script>
-      rudderanalytics=window.rudderanalytics=[];
-      for(var methods=["load","page","track","identify","alias","group","ready","reset","getAnonymousId","setAnonymousId"],i=0;i<methods.length;i++){
-        var method=methods[i];
-        rudderanalytics[method]=function(a){
-          return function(){
-            rudderanalytics.push([a].concat(Array.prototype.slice.call(arguments)))
-          }
-        }(method)
-      }
+    <script type="text/javascript">
+      !function(){"use strict";window.RudderSnippetVersion="3.0.14";var e="rudderanalytics";window[e]||(window[e]=[])
+      ;var rudderanalytics=window[e];if(Array.isArray(rudderanalytics)){
+      if(true===rudderanalytics.snippetExecuted&&window.console&&console.error){
+      console.error("RudderStack JavaScript SDK snippet included more than once.")}else{rudderanalytics.snippetExecuted=true,
+      window.rudderAnalyticsBuildType="legacy";var sdkBaseUrl="https://cdn.rudderlabs.com/v3";var sdkName="rsa.min.js"
+      ;var scriptLoadingMode="async"
+      ;var t=["setDefaultInstanceKey","load","ready","page","track","identify","alias","group","reset","setAnonymousId","startSession","endSession","consent"]
+      ;for(var r=0;r<t.length;r++){var n=t[r];rudderanalytics[n]=function(t){return function(){var r
+      ;Array.isArray(window[e])?rudderanalytics.push([t].concat(Array.prototype.slice.call(arguments))):null===(r=window[e][t])||void 0===r||r.apply(window[e],arguments)
+      }}(n)}try{new Function('return import("")'),window.rudderAnalyticsBuildType="modern"}catch(a){}
+      var i=document.head||document.getElementsByTagName("head")[0]
+      ;var d=document.body||document.getElementsByTagName("body")[0];window.rudderAnalyticsAddScript=function(e,t,r){
+      var n=document.createElement("script");n.src=e,n.setAttribute("data-loader","RS_JS_SDK"),t&&r&&n.setAttribute(t,r),
+      "async"===scriptLoadingMode?n.async=true:"defer"===scriptLoadingMode&&(n.defer=true),
+      i?i.insertBefore(n,i.firstChild):d.insertBefore(n,d.firstChild)},window.rudderAnalyticsMount=function(){
+      "undefined"==typeof globalThis&&(Object.defineProperty(Object.prototype,"__globalThis_magic__",{get:function get(){
+      return this},configurable:true}),__globalThis_magic__.globalThis=__globalThis_magic__,
+      delete Object.prototype.__globalThis_magic__),
+      window.rudderAnalyticsAddScript("".concat(sdkBaseUrl,"/").concat(window.rudderAnalyticsBuildType,"/").concat(sdkName))
```

Comment:
It is better to have this also.
Can you update the README file to replace this value with the actual write key?

```suggestion
      window.rudderAnalyticsAddScript("".concat(sdkBaseUrl,"/").concat(window.rudderAnalyticsBuildType,"/").concat(sdkName), "data-rsa-write-key", "<WRITE_KEY>")
```

Reply:
I've added the proposed changes in the `index.html` file. But what exactly do I need to update in the readme?

Reply:
Let's add a step to replace this placeholder write key with the actual value.

Reply:
Are you suggesting replacing the placeholder writeKey in the `index.html` itself, right? If that's the case then I need to research it.

Reply:
https://github.com/rudderlabs/rudder-sdk-flutter/blob/develop/packages/example/README.md

```
2. To deploy the "web" package, ensure to replace the `<WRITE_KEY>` in `/web/index.html` with the actual write key value.
```